language-design.md

```diff
@@ -17,6 +17,25 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+Our first quote:
+
+> In a race, a small language with warts will beat a well designed
+> language because users will not wait for the right thing; they will use the language that
+> is quick and cheap, and put up with the warts. [Steele 1998]
+
+We thought this was an interesting idea - Steele’s argument against small languages was straightforward, but we were not as convinced by this. There are certainly situations where you’ll need this kind of speed in putting out a language, but for a DSL there is not necessarily a big market of competitors that you need to beat to the punch. If there are no competitors then we thought it might make more sense to be slower and get rid of some of the warts before releasing a language. We also thought the idea of warts was interesting, and agreed with the idea that incremental design starting with a ‘warty’ language is a good solution for a lot of languages. However, this depends on the severity of the warts because some problems are less easily fixable after the language is released.
+
```

Comment:
You hit on a key idea--not all problems are created equal. Assuming that the "warts" aren't fatal design flaws, having a few to start out isn't alll bad. However, we shouldn't be totally careless in our design for the sake of producing a language quickly.